Wikipedia/Code/WMFArticleFooterMenuViewController.m

```diff
@@ -39,6 +39,16 @@ - (void)viewWillAppear:(BOOL)animated {
     [self.tableView deselectRowAtIndexPath:[self.tableView indexPathForSelectedRow] animated:YES];
 }
 
+- (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSInteger)section {
```

Comment:
> would this also work if we set sectionHeaderHeight and sectionFooterHeight properties in viewDidLoad

No unfortunately that doesn't seem to work...

![screen shot 2015-12-23 at 11 45 54 am](https://cloud.githubusercontent.com/assets/3143487/11983767/e2ac0d7c-a96a-11e5-938d-cbce8d2cb053.png)